package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "crud-api",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint src",
+    "start:dev": "cross-env NODE_ENV=development nodemon --exec ts-node-dev ./src/server.ts",
+    "start:prod": "cross-env NODE_ENV=production node ./dist/server.js",
+    "start:multi": "cross-env NODE_ENV=production npm run build && cross-env USE_CLUSTER=true npm run start",
+    "build": "webpack",
+    "prestart:prod": "npm run build",
+    "start": "npm run start:dev",
+    "test": "jest"
+  },
+  "repository": {
+    "type": "git",
+    "url": "crud-api"
+  },
+  "author": "aliceinapple",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/eslint": "^8.40.2",
+    "@types/jest": "^29.5.2",
+    "@types/nodemon": "^1.19.2",
+    "@types/uuid": "^9.0.2",
+    "@typescript-eslint/eslint-plugin": "^5.60.1",
+    "@typescript-eslint/parser": "^5.60.1",
+    "cross-env": "^7.0.3",
+    "dotenv": "^16.3.1",
+    "eslint": "^8.44.0",
+    "jest": "^29.5.0",
+    "nodemon": "^2.0.22",
+    "prettier": "^2.8.8",
+    "ts-jest": "^29.1.1",
+    "ts-loader": "^9.4.4",
+    "ts-node": "^10.9.1",
+    "ts-node-dev": "^2.0.0",
+    "typescript": "^5.1.6",
+    "uuid": "^9.0.0",
+    "webpack": "^5.88.1",
+    "webpack-cli": "^5.1.4"
+  },
+  "dependencies": {
+    "cluster": "^0.7.7"
```

Comment:
You shouldn't have done this. Node.js contains a built-in `Cluster API` package. I won't reduce your points on this because this package does the exact same thing as the API.